app/models/category.rb

```diff
@@ -2,7 +2,8 @@ class Category < ApplicationRecord
   belongs_to :user, dependent: :destroy
   has_many :activities_categories
   has_and_belongs_to_many :activities
-  validates_presence_of :name, :icon
+  validates_presence_of :name
+  validates :icon, url: true
 
   def total_amount
     activities.sum(:amount)
```

Comment:
- [ ] After adding the image URL, I am getting the following error, kindly fix it to make it run as well as use your project.

![image](https://user-images.githubusercontent.com/55635977/160148281-fc4eaab0-84a1-42d8-9831-9d7667bf3373.png)

Reply:
fixed. Thank you